docs_sphinx/user/nmlimport.rst

```diff
@@ -0,0 +1,322 @@
+NeuroML importer (nmlimport library)
+================
```

Comment:
The underline should be the full length of the title (but actually not sure that we need the "nmlimport library" part her).